src/System.Memory/tests/ParsersAndFormatters/Formatter/FormatterTestData.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Text;
+using System.Numerics;
+using System.Buffers.Text;
+using System.Globalization;
+using System.Collections.Generic;
+
+using Xunit;
+
+namespace System.Buffers.Text.Tests
```

Comment:
Do we have code coverage numbers?

Reply:
Utf8Formatter: 100%

FormattingHelper: 100%

Utf8Parser: 98.8% (the 1.2% are NotImplementedException and Debug.Assert paths.)

ParserHelper: 100%

MutableDecimal: 100%

System.Number: 98.2% (the 1.8% are paths that are unreachable in System.Memory but may be reachable in ProjectN where this code came from. I did not bother to analyze the much larger Project N codebase and I don't want to introduce unnecessary diffs here.)

System.NumberBuffer: 45.4% (the remaining 55.6% is the ToString() method which only exists so that NumberBuffer displays nicely in the debugger.)

StandardFormat: 100%